README.md

```diff
@@ -18,210 +18,294 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
-mkdir -p build
-cd build
+sudo apt update && sudo apt upgrade
 
-# Run CMake to generate make files
-cmake ..
+sudo apt install build-essential
 
-# To build utilizing multiple cores, append `-j` and the amount of cores to utilize, for example `cmake --build . --config Release -j8'
-cmake --build . --config Release
-```
+sudo apt install gcc
 
-### MacOS builds
-Ensure `cmake`, `zlib` and `open ssl` are installed as well as a compiler (e.g `clang` or `gcc`).
+sudo apt install zlib1g-dev
 
-In the repository root folder run the following. Ensure -DOPENSSL_ROOT_DIR=/path/to/openssl points to your openssl install location
-```bash
-# Create the build directory, preserving it if it already exists
-mkdir -p build
-cd build
+sudo apt install libssl-dev
 
-# Run CMake to generate build files
-cmake .. -DOPENSSL_ROOT_DIR=/path/to/openssl
+sudo apt install openssl
 
-# Get cmake to build the project. If make files are being used then using make and appending `-j` and the amount of cores to utilize may be preferable, for example `make -j8`
-cmake --build . --config Release
+sudo apt install mariadb-server
+
+# On some systems, this package does not exist.
+sudo apt install zlib
 ```
 
-### Windows builds (native)
-Ensure that you have either the [MSVC](https://visualstudio.microsoft.com/vs/) or the [Clang](https://github.com/llvm/llvm-project/releases/) (recommended) compiler installed. You will also need to install [CMake](https://cmake.org/download/). Currently on native Windows the server will only work in Release mode.
+#### Windows build packages
+Ensure that you have either the [MSVC](https://visualstudio.microsoft.com/vs/) (recommended) or the [Clang](https://github.com/llvm/llvm-project/releases/) compiler installed.
 
-#### Build the repository
-```batch
-:: Create the build directory
-mkdir build
-cd build
+#### MacOS build packages
+Ensure you have [brew](https://brew.sh) installed.
+You will need to install the following packages
+```bash
+brew install cmake
 
-:: Run CMake to generate make files
-cmake ..
+brew install zlib
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
-```
-#### Windows for ARM has not been tested but should build by doing the following
-```batch
-:: Create the build directory
-mkdir build
-cd build
+brew install openssl
 
-:: Run CMake to generate make files
-cmake .. -DMARIADB_BUILD_SOURCE=ON
+brew install gcc
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
+brew install mariadb
 ```
 
-### Windows builds (WSL)
-This section will go through how to install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) and building in a Linux environment under Windows. WSL requires Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11.
-
-#### Open the Command Prompt application with Administrator permissions and run the following:
-```bash
-# Installing Windows Subsystem for Linux
-wsl --install
-```
+### Database
```

Comment:
Maybe we could get rid of the subheadings under install dependencies entirely, and instead create a new section like "start database"? Also, don't you need to create an initial empty database?